operators/portworx-essentials/1.7.1/portworxoperator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,569 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: portworx-essentials.v1.7.1
+  namespace: placeholder
+  annotations:
+    olm.skipRange: '=1.7.0'
+    capabilities: Auto Pilot
+    categories: "Storage"
+    description: Free forever cloud native storage solution
+    containerImage: portworx/px-operator:1.7.1
+    repository: https://github.com/libopenstorage/operator
+    createdAt: 2022-02-11T08:00:00Z
+    support: Portworx, Inc
+    certified: "false"
+    operatorframework.io/initialization-resource: |-
+      {
+        "apiVersion": "core.libopenstorage.org/v1",
+        "kind": "StorageCluster",
+        "metadata": {
+          "name": "portworx",
+          "annotations": {
+            "portworx.io/is-openshift": "true"
+          }
+        }
+      }
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "core.libopenstorage.org/v1",
+          "kind": "StorageCluster",
+          "metadata": {
+            "name": "portworx",
+            "namespace": "test-operator",
+            "annotations": {
+              "portworx.io/is-openshift": "true"
+            }
+          },
+          "spec": {}
+        },
+        {
+          "apiVersion": "core.libopenstorage.org/v1",
+          "kind": "StorageNode",
+          "metadata": {
+            "name": "example",
+            "namespace": "test-operator"
+          },
+          "spec": {}
+        }
+      ]
+spec:
+  displayName: Portworx Essentials
+  version: 1.7.1
+  minKubeVersion: "1.12.0"
+  maturity: stable
+  replaces: portworx-essentials.v1.6.1
```

Comment:
@piyush-nimbalkar, I think you want `portworx-essentials.v1.7.0` entry here.

Reply:
@J0zi  I actually want to skip the 1.7.0 version. I have even added the `olm.skipRange: '=1.7.0'` annotation in the CSV. I did the same for the other [certified portworx operator](https://github.com/redhat-openshift-ecosystem/certified-operators/blob/main/operators/portworx-certified/1.7.1/manifests/portworx-certified.clusterserviceversion.yaml#L56).

Reply:
@J0zi @mvalarh can someone please take a look at this?

Reply:
If you will do `replaces: portworx-essentials.v1.7.0` ol will skip it for you because of `olm.skipRange: '=1.7.0'`

Reply:
@piyush-nimbalkar ^^